grpcurl.go

```diff
@@ -225,16 +225,16 @@ func ListMethods(source DescriptorSource, serviceName string) ([]string, error)
 	if err != nil {
 		return nil, err
 	}
-	if sd, ok := dsc.(*desc.ServiceDescriptor); !ok {
+	sd, ok := dsc.(*desc.ServiceDescriptor)
+	if !ok {
 		return nil, notFound("Service", serviceName)
-	} else {
-		methods := make([]string, 0, len(sd.GetMethods()))
-		for _, method := range sd.GetMethods() {
-			methods = append(methods, method.GetName())
-		}
-		sort.Strings(methods)
-		return methods, nil
 	}
+	methods := make([]string, 0, len(sd.GetMethods()))
+	for _, method := range sd.GetMethods() {
+		methods = append(methods, method.GetName())
+	}
+	sort.Strings(methods)
+	return methods, nil
```

Comment:
In general, I kind of dislike a linter forcing you to do this; if you really don't need `sd` beyond the scope of the else statement.